### PR TITLE
Update project with latest plugin best practices

### PR DESCRIPTION
**Pull request by justinmayer**
These changes update the project to be more closely in line with changes to the canonical [Cookiecutter Pelican plugin template](https://github.com/getpelican/cookiecutter-pelican-plugin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +11,18 @@ Mau Reader is a Pelican plugin that converts the [Mau](https://github.com/Projec
 
 This plugin requires:
 
-* Python 3.6+
+* Python 3.8+
 * Pelican 4.5+
-* Mau 2.0+
+* Mau 3.0+
```
Comment (lgiordani) [inline README.md:16]: I think this is premature. This version of the plugin still works with Mau 2.x

**Review by lgiordani — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (justinmayer) [reply README.md]: Oops! Fixed via 06db569.

**Comment by lgiordani** [comment]
@justinmayer I'm not familiar with pdm, but I think the tests are not using the correct virtual environment. If the object `Mau` is not defined it means that the import failed

**Review by lgiordani — APPROVED** [review]

**Comment by justinmayer** [comment]
Thanks for the review. I will squash some of these commits and then merge in a moment.

**Comment by justinmayer** [comment]
FYI, I filed an issue regarding the PDM-related problem above: https://github.com/pdm-project/pdm/issues/2235

**Pull request merged**